content/graphql/guides/forming-calls-with-graphql.md

```diff
@@ -409,7 +409,7 @@ For more information on the difference between enums and strings, see the [offic
 
 There is a _lot_ more you can do when forming GraphQL calls. Here are some places to look next:
 
-* [Pagination](https://graphql.github.io/learn/pagination/)
+* [Pagination](https://graphql.org/learn/pagination/)
 * [Fragments](https://graphql.github.io/learn/queries/#fragments)
```

Comment:
While you're making these great changes, could you update this one to `https://graphql.org/learn/queries/#fragments` as well?

Reply:
Sure! I must have been tired.